components/MultiCropSelector/MultiCropSelector.adminPermissions.comp.cy.js

```diff
@@ -0,0 +1,36 @@
+import MultiCropSelector from '@comps/MultiCropSelector/MultiCropSelector.vue';
```

Comment:
Do we actually have to click the `+` button here, or is it enough to just check that the `+` button is visible?  If so, then this test and the `guestPermissions` tests can be combined into one test file.

The `behavior` test will be the one that has to click the button and ensure that it goes to the correct page (or eventually that the correct page is loaded into the popup).